themes/src/main/resources/theme/keycloak-preview/account/index.ftl

```diff
@@ -155,7 +155,7 @@
             </div>
             </#if>
 
-            <#if realm.internationalizationEnabled  && supportedLocales?size gt 1>
+            <#if false && realm.internationalizationEnabled  && supportedLocales?size gt 1>
```

Comment:
You also need to do this on line 199.

Reply:
Removed the lines as you suggested on chat

Reply:
OK. 

Realized that this means we can remove a couple of functions from WelcomePageScripts.js.  I'll just do that when I refactor locale handling in https://issues.redhat.com/browse/KEYCLOAK-12936